azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/bindings/Binding.java

```diff
@@ -101,20 +102,22 @@ public String toString() {
                 .toString();
     }
 
-    protected String getDirectionFromAttribute() {
-        if (this.direction != null) {
-            return direction.toString();
+    protected void addProperties(List<String> requiredAttributes, Object value, Method method) {
```

Comment:
I think we don't need to pass requiredAttributes as it is an member properties of Binding class